apps/signal-store-angular-demo-standalone/src/app/products/components/products-shell/products-shell.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="d-flex flex-column h-100" *ngIf="userStore.permissions()">
+    <nav class="navbar navbar-light bg-light mb-1">
+        <a class="navbar-brand">Products</a>
+        <div class="d-flex flex-grow-1 mb-2 justify-content-between mt-2">
+            <div>
+                <button
+                    *ngIf="userStore.permissions().canUpdateProducts"
+                    class="btn btn-primary btn-sm"
+                    (click)="productsFacade.newProduct()"
+                >
+                    New
+                </button>
+            </div>
+            <app-product-filter
+                [search]="productsFacade.search()"
+                (searchChanged)="productsFacade.updateSearch($event)"
+            ></app-product-filter>
+        </div>
+    </nav>
+
+    <div class="m-3 alert alert-info d-flex align-items-center" role="alert">
+        <i class="info-icon bi bi-info-circle-fill"></i>
+        <div>
+            MiniRx
+            <strong>(Redux) Store</strong>
+            is used to manage the "products" state. The "products" state becomes part of the
+            <strong>global state</strong>
+            object.
+            <br />
+            <br />
+            You can edit the products if you enable "CanUpdateProducts" in the "John Doe" user
+            settings of this demo.
+            <br />
+            Press the ALT key while saving or deleting a product to simulate an API error.
```

Comment:
Perhaps add between parentheses that this can also be achieved with the `Option` key on Mac?

Reply:
added the mac option :)